### PR TITLE
A series of minor fixes that establish a baselines scan performance of about 30MB/s.

### PR DESCRIPTION
**Pull request by aboodman**
We should wire this up to run continuously on bots. It's super super easy to regress things accidentally (as this log shows).

### CODE REVIEW

File: sample/perf/index.html [hunk 1]
```diff
@@ -56,10 +56,12 @@
     await populate(rep, opts);
     bench.reset();
     await rep.query(async (tx) => {
-      let i = 0;
+      var count = 0;
```
Comment (arv) [inline sample/perf/index.html:54]: why var?

**Review by arv — COMMENTED** [review]
LGTM

**Review by phritz — COMMENTED** [review]
LGTM

File: sample/perf/index.html (hunk 1 above)
Comment (aboodman) [reply sample/perf/index.html:54]: Because I sux. Looks like you or I already fixed this.

**Pull request merged**